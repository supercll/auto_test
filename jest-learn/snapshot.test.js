import { generateConfig, generateAnotherConfig } from "./index";

// test("generateConfig 测试配置", () => {
//   expect(generateConfig()).toMatchSnapshot({
//     time: expect.any(Date), // 排除时间格式测试
//   });
// });
// test("generateAnotherConfig 测试配置", () => {
//   expect(generateAnotherConfig()).toMatchSnapshot({
//     time: expect.any(Date),
//   });
// });

test("generateConfig 测试配置", () => {
  expect(generateConfig()).toMatchInlineSnapshot(
    {
      time: expect.any(Date)
    },
    `
    Object {
      "port": 8080,
      "server": "http://localhost",
      "time": Any<Date>,
    }
  `
  );
});
