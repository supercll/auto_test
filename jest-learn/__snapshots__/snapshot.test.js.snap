// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateAnotherConfig 测试配置 1`] = `
Object {
  "port": 8080,
  "server": "http://localhost",
  "time": "2021",
}
`;

exports[`generateConfig 测试配置 1`] = `
Object {
  "port": 8080,
  "server": "http://localhost",
  "time": Any<Date>,
}
`;
